### ISSUE TITLE
More documentation annotations in xspec.rnc

### ISSUE THREAD

**Issue by raducoravu** [xspec/xspec#113]
For example add more ## documentation before each of the attribute definitions:

                             attribute pending { text }?,
                             attribute focus { text }?

as Oxygen XML Editor will show this documentation when you want to insert each attribute.

**Comment by cirulls** [comment]
Thanks for your input, more documentation is always good!

Attributes ```pending``` and ```focus``` have exactly the same function as the elements with the same names as described in the [documentation](https://github.com/xspec/xspec/wiki/Writing-Scenarios#focusing-your-efforts).

In [xspec.rnc](https://github.com/xspec/xspec/blob/master/src/schemas/xspec.rnc) the behaviour of the attributes are documented inside ```scenario``` as follows: 

```
scenario = 
	## A scenario defines the environment in which a piece of processing takes place.
	##
	## Any parameters defined within a scenario equate to global (stylesheet)
	## parameters.
	##
	## If a scenario has a pending attribute, this has the same semantics as
	## wrapping the scenario in a <pending> element with a label equal to the value
	## of the pending attribute.
	##
	## If any scenario has a focus attribute, any scenarios without a focus 
	## attribute will be classed as pending.
	element scenario { common-scenario-attributes, shared?,
		label,
		( matching-scenario |
		  function-scenario |
		  named-scenario ) }

common-scenario-attributes = common-attributes,
                             attribute pending { text }?,
                             attribute focus { text }?
```

Is it just a matter of repeating this information before ```common-scenario-attributes```? Would that display fine in Oxygen?

**Comment by raducoravu** [comment]
The information needs to be set before each attribute definition. I will prepare a pull request.

**Comment by raducoravu** [comment]
There are a couple more attributes like "href", "select" which I do not know how to describe, my knowledge of XSpec is not that great. So they should be documented in the same way.

**Comment by cirulls** [comment]
I added documentation for attributes ```href``` and ```select``` in https://github.com/xspec/xspec/pull/115#issuecomment-292457562.

**Comment by cirulls** [comment]
This is now fixed in #115.
